CHANGELOG.md

```diff
@@ -1,73 +1,85 @@
 # Mattermost Desktop Application Changelog
 
-## UNDER DEVELOPMENT
+## Release v4.1.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost Desktop changelog appears
-in the product's `master` branch to note key changes committed to `master`
-and are on their way to the next stable release.
-When a stable release is pushed, "UNDER DEVELOPMENT" heading is removed
-from the final changelog of the release.
-
-Release date: TBD
+Release date: May 16, 2018
 
 ### Improvements
 
 #### All Platforms
-- Added "Enable GPU hardware acceleration" option. It's disabled by default for stability.
-[#734](https://github.com/mattermost/desktop/pull/734)
+ - Improved stability and performance
+   - Disabled GPU acceleration by default to reduce app crashes. It can be enabled in the App Settings. [#734](https://github.com/mattermost/desktop/pull/734)
 [#750](https://github.com/mattermost/desktop/pull/750)
-- Enabled Certificate Transparency verification in HTTPS.
+   - Reduced memory usage by periodically clearing cache. [#746](https://github.com/mattermost/desktop/issues/710)
+   - Fixed app crashing when a server tab was drag-and-dropped to the message view.
+[#667](https://github.com/mattermost/desktop/issues/667)
+   - Fixed Windows crash issues during installation. [#728](https://github.com/mattermost/desktop/issues/728)
+   - Fixed Mac and Linux crashing after toggling "Show Mattermost icon in menu bar" app setting.
+- Updated design for loading animation icon.
+[#748](https://github.com/mattermost/desktop/issues/748)
+ - Enabled Certificate Transparency verification in HTTPS.
```

Comment:
@yuya-oc Which change was this? I'm not sure I understand it

Reply:
@jasonblais It has been effective since merging #741. #743 had disabled the Certificate Transparency verification in v4.0.1 app though, I had added [a version check of Electron](https://github.com/mattermost/desktop/pull/743/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR616).

Reply:
Right, I remember now. Wondering if we need to include it in changelog?

Reply:
I think so. It improves reliability of HTTPS, but it affects connectivity if server certificates are still not adapted to Certificate Transparency.

Reply:
Yeah makes sense. Would it be appropriate to link to https://www.certificate-transparency.org/what-is-ct for those who don't know what certificate transparency is?

Reply:
Yes, it would be helpful!